packages/components/src/components/hds/popover-primitive/index.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { assert, warn } from '@ember/debug';
+import { next } from '@ember/runloop';
+import { guidFor } from '@ember/object/internals';
+import { modifier } from 'ember-modifier';
+
+import registerEvent from '../../../modifiers/hds-register-event';
+import anchoredPositionModifier from '../../../modifiers/hds-anchored-position';
+
+// https://github.com/oddbird/popover-polyfill?tab=readme-ov-file#with-npm
+// this is needed until Firefox releases the version 126 (up to 125 didn't support the Popover API)
+import {
+  // this call polyfills some of the browser methods to emulate the Popover API
+  apply as applyPopoverApiPolyfill,
+  // we'll use these two flags to overwrite the popover positioning strategy
+  // this is specifically done for Firefox (we need Firefox 126 to be released, to support the last 2 versions)
+  // see: https://whattrainisitnow.com/release/?version=126
+  isSupported as isPopoverApiSupported,
+  isPolyfilled as isPopoverApiPolyfilled,
+} from '@oddbird/popover-polyfill/fn';
+
+// we use this to re-export the values
+export {
+  PLACEMENTS,
+  DEFAULT_PLACEMENT,
+} from '../../../modifiers/hds-anchored-position';
+
+export default class HdsPopoverPrimitiveComponent extends Component {
+  @tracked isOpen = this.args.isOpen ?? false;
+  @tracked isClosing = false;
+  // this will enable "soft" events for the toggle ("hover" and "focus")
+  @tracked enableSoftEvents = this.args.enableSoftEvents ?? false;
+  // this will enable "click" events for the toggle
+  @tracked enableClickEvents = this.args.enableClickEvents ?? false;
+
```

Comment:
I believe challenged these properties need to be tracked in the previous review – any thoughts on this?

Reply:
I _think_ I made the change but then it may have been lost in some rebase. You're absolutely right, I'll remove the tracking.